tests/util.py

```diff
@@ -26,3 +31,17 @@ def import_script(name, script):
     module = importlib.util.module_from_spec(spec)
     spec.loader.exec_module(module)
     return module
+
+
+# shlex.join added in python 3.8.
```

Comment:
It's so I could keep using ubuntu-18.04 in the GitHub tests where python is 3.6. I don't recall what we have on 4.0 but I think it's newer than that. I wanted to keep supporting that if it wasn't too much effort since we tell people they can run the image builder on basically any debian style host.

However, in a real image build this code runs in the buildroot against EOS, so it's all a bit silly. For a bit I did attempt to generate a buildroot and then run pytest inside that, but it was too complicated. If we ever make the eos containers public, I'd use those in the GitHub tests but even then I'd attempt to use the appropriate branch. So, probably we could tear this and the other compat code out at some point.